satellite/satellitedb/usercredits.go

```diff
@@ -73,28 +79,54 @@ func (c *usercredits) UpdateAvailableCredits(ctx context.Context, appliedCredits
 	if err != nil {
 		return errs.Wrap(errs.Combine(err, tx.Rollback()))
 	}
+	if len(availableCredits) == 0 {
+		return errs.Wrap(tx.Commit())
+	}
+
+	var infos []updateInfo
 
 	for _, credit := range availableCredits {
-		if appliedCredits == 0 {
+		if chargedCredits == 0 {
 			break
 		}
 
-		updatedUsedCredit := credit.CreditsEarnedInCents
+		updateCredit := credit.CreditsEarnedInCents - credit.CreditsUsedInCents
```

Comment:
Should this be `updatedCredit` ?

Reply:
It's not updated yet at this point so I don't want to name it as `updated`. Maybe I will change it to `creditsForUpdate` or something like that